tests/test_images.py

```diff
@@ -1,21 +1,25 @@
 import pytest
+import uuid
 from io import BytesIO
 from aiodocker import utils
 from aiodocker.exceptions import DockerError
 
 
+def _random_name():
+    return "aiodocker-" + uuid.uuid4().hex[:7]
+
+
 @pytest.mark.asyncio
 async def test_build_from_remote_file(docker):
     remote = ("https://raw.githubusercontent.com/aio-libs/"
               "aiodocker/master/tests/docker/Dockerfile")
 
-    tag = "image:1.0"
+    tag = "%s:1.0" % _random_name()
     params = {'tag': tag, 'remote': remote, 'stream': True}
     stream = await docker.images.build(**params)
 
-    async for output in stream:
-        if "Successfully tagged image:1.0\n" in output:
-            break
+    async for _ in stream:  # noqa: F841
```

Comment:
That's the funny part of this PR. If you don't loop, you're image will not finish to build and the code will break. Maybe that only happens if the `stream=True` is active though. I know it's the case for docker-py and I just checked with a big image with aiodocker and it behaves the same.